draft-bonaventure-mptcp-converters.mkd

```diff
@@ -671,6 +685,14 @@ own value. For the TCP Options that are included in the 'TCP Options' field with
 an optional value, it shall copy the entire option for use in the connection with the destination peer. This feature is required to support TCP Fast Open.
 
 
+### External Connect TLV {#sec-extrnl-connect}
+The External Connect TLV is used when a Remote Host initiates a connection with
+an internal Host. This TLV is exactly the same as {{fig-connect}} with the only
```

Comment:
The TLV should include a lifetime in both the request/response. 
The TLV must include the assigned external IP address/port in the response.

Reply:
Hello, 
Thanks for your comments.  I wanted to keep the support for incoming     
connection as simple and generic as possible.  By introducing lifetime, port    
numbers in the request and external (IPaddress , port) in the response, We are  
essentially reimplementing NAT, which I believe, makes the system more          
complicated. What I had initially in mind is that there is connectivity between 
'internal Host'  and TC. There may also be connectivity between 'external Host' 
and 'internal Host'.  The main reason (possibly more) that an external host   
may connect to an internal host through the TC is that there is no direct       
connectivity between the external host and internal host.  With this scenario,  
The idea was to enable the support for incoming connection functionality        
'symmetric' to the way Connect requests are handled by the TC. This also results
in TC having to maintain less state.